specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-10-01/updates.json

```diff
@@ -0,0 +1,387 @@
+          "description": "Description of the update.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "state": {
+          "description": "State of the update as it relates to this stamp.",
+          "type": "string",
+          "enum": [
+            "HasPrerequisite",
+            "Obsolete",
+            "Ready",
+            "NotApplicableBecauseAnotherUpdateIsInProgress",
+            "Preparing",
+            "Installing",
+            "Installed",
+            "PreparationFailed",
+            "InstallationFailed",
+            "Invalid",
+            "Recalled",
+            "Downloading",
+            "DownloadFailed",
+            "HealthChecking",
+            "HealthCheckFailed",
+            "ReadyToInstall",
+            "ScanInProgress",
+            "ScanFailed"
+          ],
+          "x-ms-client-flatten": true,
+          "x-ms-enum": {
+            "name": "state",
+            "modelAsString": true
+          }
+        },
+        "prerequisites": {
+          "description": "If update State is HasPrerequisite, this property contains an array of objects describing prerequisite updates before installing this update. Otherwise, it is empty.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UpdatePrerequisite"
+          },
+          "x-ms-identifiers": []
+        },
+        "packagePath": {
+          "description": "Path where the update package is available.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "packageSizeInMb": {
+          "description": "Size of the package. This value is a combination of the size from update metadata and size of the payload that results from the live scan operation for OS update content.",
+          "type": "number",
+          "x-ms-client-flatten": true
+        },
+        "displayName": {
+          "description": "Display name of the Update",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "version": {
+          "description": "Version of the update.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "publisher": {
+          "description": "Publisher of the update package.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "releaseLink": {
+          "description": "Link to release notes for the update.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "availabilityType": {
+          "description": "Indicates the way the update content can be downloaded.",
+          "type": "string",
+          "enum": [
+            "Local",
+            "Online",
+            "Notify"
+          ],
+          "x-ms-client-flatten": true,
+          "x-ms-enum": {
+            "name": "availabilityType",
+            "modelAsString": true
+          }
+        },
+        "packageType": {
```

Comment:
Does this have a fixed set of values? If so, it should be modeled as a string enum.

Reply:
Values are not fixed